var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -13,25 +13,36 @@
 _CORENRN_MODLIST_FNAME = "coreneuron_modlist.txt"
 _BUILD_NEURODAMUS_FNAME = "build_neurodamus.sh"
 
+PYNEURODAMUS_DEFAULT_V = '2.9.0'
```

Comment:
Maybe we should do the same for NEURODAMUSCORE_DEFAULT_V? I guess we would need it for a bit

Reply:
I don't think we need that anymore, I only left one previous version depending on core